lib/msal-node/src/client/IConfidentialClientApplication.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthenticationResult, AuthorizationCodeRequest, AuthorizationUrlRequest, ClientCredentialRequest, Logger, OnBehalfOfRequest, RefreshTokenRequest, SilentFlowRequest } from "@azure/msal-common";
+import { NodeConfigurationAuthError } from "../error/NodeConfigurationAuthError";
+import { TokenCache } from "../cache/TokenCache";
+export interface IConfidentialClientApplication {
```

Comment:
Should be exported in `index.ts` as well

Reply:
Done.